src/components/AboutUs/Members/Members.jsx

```diff
@@ -1,23 +1,43 @@
-import React from "react";
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+import Member from "./Member";
 import { MEMBERS_TITLE } from "../../../constants/AboutUs";
-import MEMBERS from "../../../mockup-db/aboutUsMembers";
 import "./Members.scss";
+import SpinnerWrapper from "../../../containers/spinner";
 
-const Members = () => (
-  <div className="about-members">
-    <h3 className="members-title">{MEMBERS_TITLE}</h3>
-    <div className="members-container">
-      {MEMBERS.map(({ name, description, img }) => (
-        <div className="member" key={name}>
-          <img className="member__image" src={img} alt={name} />
-          <div className="member__info">
-            <h2 className="member__name">{name}</h2>
-            <p className="member__description">{description}</p>
-          </div>
+const Members = ({ members, loading, error, getMembers }) => {
+  useEffect(() => {
+    if (members.length === 0) {
```

Comment:
Move to container `members.length === 0`

Reply:
done